IGNORE.py

```diff
@@ -0,0 +1,87 @@
+import turtle, random
+
+def draw_sunflower(t):
+    for _ in range(36):
+        t.circle(50)
+        t.right(10)
+
+def draw_rose(t):
+    for _ in range(5):
+        t.circle(60, 180)
+        t.left(144)
+
+def draw_daisy(t):
+    for _ in range(12):
+        t.circle(40)
+        t.left(30)
+
+def draw_tulip(t):
+    t.begin_fill()
+    for _ in range(2):
+        t.circle(40, 90)
+        t.circle(40 // 2, 90)
+    t.end_fill()
+
+def draw_lily(t):
+    for _ in range(5):
+        t.circle(100, 60)
+        t.left(144)
+
+def get_user_input():
+    flower_types = {
+        '1': 'sunflower',
+        '2': 'rose',
+        '3': 'daisy',
+        '4': 'tulip',
+        '5': 'lily'
+    }
+
+    num_flowers = int(input("How many flowers would you like to draw? "))
+
+    print("Choose a type of flower:")
+    print("1. Sunflower")
+    print("2. Rose")
+    print("3. Daisy")
+    print("4. Tulip")
+    print("5. Lily")
+    flower_choice = input("Enter the number corresponding to the flower: ")
+
+    return num_flowers, flower_types[flower_choice]
+
```

Comment:
@RohitDen Change the code segment to take in a sentence from the user and gather details about what to draw.

Reply:
import re

flower_types = {
    'sunflower': 'sunflower',
    'rose': 'rose',
    'daisy': 'daisy',
    'tulip': 'tulip',
    'lily': 'lily'
}

def get_flower_input(sentence):
    sentence = sentence.lower()
    num_flowers = re.findall(r'\d+', sentence)
    num_flowers = int(num_flowers[0]) if num_flowers else 1

    picked_flowers = []
    for flower in flower_types:
        if flower in sentence:
            picked_flowers.append(flower_types[flower])

    return num_flowers, picked_flowers
    

user_input = input("Please Describe the flower you want the program to draw: ( For example, I want 3 Roses ) ")
num_flowers, picked_flowers = get_flower_input(user_input)

print(f"Number of flowers to draw: {num_flowers}")
print("Flowers to draw:")

for flower in picked_flowers:
    print(f"- {flower.capitalize()}")

if not picked_flowers:
    print("No flower type detected.")